README.md

```diff
@@ -40,7 +40,11 @@ pip install pyElli[fitting]
 ```
 
 This installs pyElli with the additional fitting capabilities and interactive widgets.
-If don't want to have this functionality just drop the `[fitting]` in the end.
+If you don't want to have this functionality just drop the `[fitting]` in the end.
+
+To increase performance of the 4x4 Solver by a factor of 20, it is recommended to
```

Comment:
Maybe it makes more sense to point to scipy's solver being a bit slower, leaving the factor of 20 out (since this might also depend on arch) and also adding a link to the discussion (but maybe thats a bit overwhelming in the readme).